docs/content.go

```diff
@@ -701,11 +701,13 @@ Enterprise Performance Computing (EPC)`
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// Run-help
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-	RunHelpUse   string = `run-help <image path>`
-	RunHelpShort string = `Show the user-defined help for an image`
+	RunHelpUse   string = `singularity run-help [run-help options] <image path>`
```

Comment:
Cobra adds the `singularity` bit, like this it will display, singularity 2 times

Reply:
oops, yes

Reply:
;)